tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_api.cc

```diff
@@ -0,0 +1,957 @@
+/**
+ * @file        unittest_tizen_nnfw_runtime_api.cc
+ * @date        07 Oct 2019
+ * @brief       Unit test for Tizen nnfw tensor filter plugin.
```

Comment:
It's minor comment. I think we now need more specified `brief` comments in both `unittest_tizen_nnfw_runtime_api.cc` and `unittest_tizen_nnfw_runtime_raw.cc`

Reply:
updated